sunkit_pyvista/plotter.py

```diff
@@ -119,20 +128,34 @@ def _pyvista_mesh(self, m):
         grid['data'] = m.plot_settings['norm'](data)
         return grid
 
-    def plot_map(self, m, **kwargs):
+    @u.quantity_input
+    def plot_map(self, m, clip_interval: u.percent = None, **kwargs):
         """
         Plot a map.
 
         Parameters
         ----------
         m : `sunpy.map.Map`
             Map to be plotted.
+        clip_interval : two-element `~astropy.units.Quantity`, optional
+            If provided, the data will be clipped to the percentile
+            interval bounded by the two numbers.
         **kwargs :
             Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
         """
         cmap = kwargs.pop('cmap', m.cmap)
         mesh = self._pyvista_mesh(m)
-        self.plotter.add_mesh(mesh, cmap=cmap, **kwargs)
+
+        if clip_interval is not None:
+            if len(clip_interval) == 2:
+                clim = self._get_clim(data=mesh['data'],
```

Comment:
Codecov says this line isn't covered?

Reply:
Ah, I forgot to add this in. I'll fix it